### PR TITLE
Fix spelling typo in Live branch

### PR DESCRIPTION
**Pull request by jamesmontemagno**
Fixes type in live via merge issues earlier

### CODE REVIEW

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Samples/Samples/ViewModel/DeviceInfoViewModel.cs](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/Samples/Samples/ViewModel/DeviceInfoViewModel.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceDisplay/DeviceDisplay.shared.cs](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/Xamarin.Essentials/DeviceDisplay/DeviceDisplay.shared.cs) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-android.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/FrameworksIndex/xamarin-essentials-android.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-ios.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/FrameworksIndex/xamarin-essentials-ios.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-uwp.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/FrameworksIndex/xamarin-essentials-uwp.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/FrameworksIndex/xamarin-essentials.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/EnergySaverStatusChangedEventArgs.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/Xamarin.Essentials/EnergySaverStatusChangedEventArgs.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/EnergySaverStatusChangedEventHandler.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/Xamarin.Essentials/EnergySaverStatusChangedEventHandler.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/ScreenMetricsChangedEventArgs.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/Xamarin.Essentials/ScreenMetricsChangedEventArgs.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/ScreenMetricsChangedEventHandler.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/Xamarin.Essentials/ScreenMetricsChangedEventHandler.xml) | :white_check_mark:Succeeded | |
[docs/en/index.xml](https://github.com/xamarin/Essentials/blob/bug/fix-ScreenMetricsChanagedEventArgs/docs/en/index.xml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C23%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201807231812441346-403%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NPVaWBWGwJN%2F9iZKeJkw8swFzFxpQlgY8Z%2BN7EClAW0%3D&st=2018-07-23T18%3A09%3A01Z&se=2018-08-23T18%3A14%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jamesmontemagno** [comment]
build

File: docs/en/index.xml [hunk 1]
```diff
@@ -127,7 +127,7 @@
       <Type Name="Preferences" Kind="Class" />
       <Type Name="ScreenLock" Kind="Class" />
       <Type Name="ScreenMetrics" Kind="Structure" />
-      <Type Name="ScreenMetricsChanagedEventArgs" Kind="Class" />
+      <Type Name="ScreenMetricsChangedEventArgs" Kind="Class" />
       <Type Name="ScreenMetricsChanagedEventHandler" Kind="Delegate" />
```
Comment (Redth) [inline docs/en/index.xml]: Missed one.

**Comment by jamesmontemagno** [comment]
Oh, i see in 0.8.0 this was actually incorrect and it is solved in master!

**Pull request closed without merging**